emacs.fnl

```diff
@@ -152,11 +152,15 @@
                                 (: app :activate)
                                 (windows.maximize-window-frame (: modal :machine)))))))})))
 
-{:enable-edit-with-emacs                 enable-edit-with-emacs
- :disable-edit-with-emacs                disable-edit-with-emacs
- :add-state                              add-state
- :edit-with-emacs                        edit-with-emacs
- :switchToApp                            switch-to-app
- :switchToAppAndPasteFromClipboard       switch-to-app-and-paste-from-clipboard
- :editWithEmacsCallback                  edit-with-emacs-callback
- :add-app-specific                       add-app-specific}
+{:add-app-specific                 add-app-specific
+ :add-state                        add-state
```

Comment:
Do we still need this add-state + export and to export the editWihEmacsCallback?

Reply:
Yeah, not sure. I'm still digging through your implementation of the state machine bits, there are definitely things that may not be needed anymore.

Reply:
Fair enough, I'll be writing some more docs tonight.